raco/algebra.py

```diff
@@ -381,6 +381,146 @@ def shortStr(self):
 """Logical Relational Algebra"""
 
 
+class ScanIDB(ZeroaryOperator):
```

Comment:
I think it's because that the plan would have cycles if `IDBController` is the "child", then RACO would apply rules endlessly.